### ISSUE TITLE
http://www.miniminiplus.pl/ - anti-adblocker

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardFilters#3884]
http://www.miniminiplus.pl/filmy/pokaz/wissper-cisza-i-spokoj

![screen shot 2016-12-26 at 12 19 41 pm](https://cloud.githubusercontent.com/assets/15856304/21480443/b09a3e44-cb65-11e6-8903-ae8a6fc799a1.png)

<details>

![screen shot 2016-12-26 at 12 20 58 pm](https://cloud.githubusercontent.com/assets/15856304/21480450/c7caa810-cb65-11e6-8338-bb40234f392d.png)

**Comment by Alex-302** [comment]
Added to English filter
```
@@||miniminiplus.pl$generichide
```

**Comment by evgeniy-ADG** [comment]
It's still blocked on Android for some reason.

<details>

![screenshot_20161229-221026](https://cloud.githubusercontent.com/assets/15856304/21553899/e185a65a-ce13-11e6-80b3-69b6355deeb2.png)


</details>

http://www.miniminiplus.pl/filmy/pokaz/stacyjkowo-male-ciuchciaki-odwazny-bruno-odc-20

[state (83).txt](https://github.com/AdguardTeam/AdguardFilters/files/677818/state.83.txt)

**Comment by Alex-302** [comment]
Added
```
@@||tvn.adocean.pl/ad.xml$domain=miniminiplus.pl
```

**Comment by evgeniy-ADG** [comment]
The customer states that the problem persists... It's fine at my end.

[debug (8).txt](https://github.com/AdguardTeam/AdguardFilters/files/692842/debug.8.txt)
[state (96).txt](https://github.com/AdguardTeam/AdguardFilters/files/692843/state.96.txt)

http://www.miniminiplus.pl/filmy/pokaz/barbie-dreamtopia

**Comment by Alex-302** [comment]
Also not reproduced.

**Comment by zebrum** [comment]
confirm the problem on my android device

**Comment by Alex-302** [comment]
Please check this rule:
```
@@||tvn.adocean.pl^*/ad.xml$domain=miniminiplus.pl
```

**Comment by seba2282** [comment]
Hello, 
@@||tvn.adocean.pl^*/ad.xml$domain=miniminiplus.pl 

doesn't work in user's filtering or exclussion https as prefer whitelist domain.

**Comment by Alex-302** [comment]
@seba2282 
please check
```
@@||tvn.adocean.pl/ad.xml$domain=miniminiplus.pl
```

**Comment by seba2282** [comment]
Where add it ? 
Whitelist domain https and doesn't work - scenerio one.
User filters and doesn't work two - scenerio two.
If I disable DNS filtering it will work - scenerio three.

All whitelist on main settings www.miniminiplus.pl.

I forgot. On forums, the people write about problems and they are more sites which will base on the same anti adblock. If it will help, example sites type TVN, TVN Turbo, etc don't work. They base on player.pl by TVN SA

**Comment by seba2282** [comment]
Now filtering DNS + whitelist www.miniminiplus.pl + user filter @@||adocean.pl = working

Closed :)

**Comment by seba2282** [comment]
Sorry. After added this filter generally adquard ignores blocking ads. So it is worst method. So my problem is actual.

**Comment by Alex-302** [comment]
> Where add it ?

eng filter

Remove your rules and update filters. Then check it in Chrome.

**Comment by seba2282** [comment]
Doesn't work on android chrome and active DNS filtering. All time still problem. :(

**Comment by Alex-302** [comment]
damn, try without dns)

**Comment by Alex-302** [comment]
and what DNS are you using?

**Comment by zebrum** [comment]
without dns it works fine

**Comment by seba2282** [comment]
I use dns filtering. Without DNS works. But what with it, without DNS , Ads type your android is infected (site ala google) or update extensions attacks me. With DNS , this shits are blocked. Sorry for "shits" :(

**Comment by Alex-302** [comment]
Please write your DNS IP addresses

**Comment by seba2282** [comment]
My IP to DNS prefer DNS in Adquard  ?
All works without DNS filtering.
With DNS filtering doesn't work.

**Comment by Alex-302** [comment]
Oh, I understand:) DNS filtering setting, not Adguard DNS:)
Try to add to user filter:
```
@@||tvn.adocean.pl
```

**Comment by seba2282** [comment]
Works now. Without ip adress dns and without whitelist. I am testing now.

**Comment by Alex-302** [comment]
This rule is for user filter only.

**Comment by seba2282** [comment]
I have a question. Must I still add all domain @@||tvn.adocean.pl and unblock all ads or is it a way for adding separetly exclussion site miniminiplus.pl to whitelist with dns filtering enabled ? In extension for browsers I added only site example www.miniminiplus.pl to whitelist and it worked but for Android Adquard with dns filtering doesn't work.

**Comment by Alex-302** [comment]
When DNS filtering is enabled, it is impossible to block/unblock specific URL. Domain is blocked completely. You can disable DNS filtering in Adguard settings.
Do you see ads with this rule?

**Comment by seba2282** [comment]
Yes, example site cdax.tv will show me ads type "upgrade android", "update plugin" or "remove virus" in new window. This ads show as window google and they are hard to disabling.If I have dns filtering wothout this rule, I can't see any ads this type. But miniminiplus.pl doesn't work without this rule. Did it help, what I wrote ?

**Comment by seba2282** [comment]
Sorry my bug. "without" not "wothout".

**Comment by Alex-302** [comment]
Unblocking in filters leads to unblocking ads on many sites.
So you are using Adguard DNS+Adguard app, or enabled DNS filtering in Adguard app?
I canl try to exclude blocking rule from dns filter.

**Comment by seba2282** [comment]
I use android app Adquard + chrome + filtering dns.

**Comment by Alex-302** [comment]
`filtering dns`
do you mean an option in Adguard? If `yes`, you can disable it.

**Comment by seba2282** [comment]
Yes, I write about option in Adquard. If I disable it, I see ads type remove virus, upgrade Android, update plugin, etc. If enable it, I don't see this ads.

**Comment by Alex-302** [comment]
> I see ads type remove virus, upgrade Android, update plugin, etc.
Try with simplified domains filter.
Where did you get that ads?

**Comment by seba2282** [comment]
I have active all filter without useful advertising ads. This ads were on example cdax.tv in new window.

**Comment by Alex-302** [comment]
Please open new issue about it.
